distributions/README.md

```diff
@@ -0,0 +1,20 @@
+# Criteria for Supported Distributions
+
+Support for a distribution implies all the duties covered by the Approver and Maintainer role requirements. In addition, support means that the Collector SIG is the owner and maintainer of the binaries/images of the different Collector distributions and is responsible for the pipeline that produces those artifacts.
+
+The Collector SIG must support at least one distribution that is recommended for production which includes support for Prometheus, Jaeger, Zipkin, and OpenCensus.
+
+Distributions supported by the Collector SIG should fulfill the following criteria:
+
+1. Serve a specific purpose and those purposes should have minimal overlap.
+2. Meet general needs and not be too niche.
+3. Only target the needs of the OpenTelemetry project.
+4. May be focused on development and proof of concept use cases.  The distribution should clearly indicate whether the Collector SIG recommends the distribution be used in Production environments.
+5. Must only include components from the `opentelemetry-collector` and `opentelemetry-collector-contrib` repositories.
+6. Have a clearly defined list of criteria for which components are included.
+7. Must include the following assets except where the specific purpose of the distribution is naturally associated with a subset of these assets. In such cases, it should be clearly stated which assets are skipped and why.  Additional asset may be included if the distro desires:
```

Comment:
> Additional asset may be included if the distro desires

Who decides what is desirable? Maybe it should say something along the lines of "if the maintainers agree"

Reply:
Good idea.

Reply:
Updated